agent/php_internal_instrument.c

```diff
@@ -768,6 +770,7 @@ static void nr_php_instrument_datastore_operation_call(
     nr_datastore_t datastore,
     const char* operation,
     nr_datastore_instance_t* instance,
+    bool instance_only,
```

Comment:
This just feels really awkward that `nr_php_instrument_datastore_operation_call` is used solely to create instance metrics. The name of this function strongly suggests that it is interested in datastore operation, whereas the `bool` flag forces this suggestion to be discarded. For what this PR tries to do, I'd expect new function (or functions): `nr_php_create_datastore_instance_metric_call` (`nr_php_create_datastore_instance_metric`) and this function remain unchanged. Current implementation seems to work, however reading the code requires additional mental power.